share/smack/include/smack.di

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Please add license info - you can simply copy-paste it from almost any other file.